<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RetimeDialog</class>
 <widget class="QDialog" name="RetimeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Retime Animation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>&lt;h1&gt;Retime Animation&lt;/h1&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descLabel">
     <property name="text">
      <string>Retiming allows you to adjust the FPS and playback speed of your project independently. It accomplishes this by moving the frames closer together or further apart to reach the desired speed and frame rate.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="frameRateLabel">
     <property name="text">
      <string>&lt;span style=&quot; text-decoration: underline;&quot;&gt;Frame Rate&lt;/span&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fpsGroup">
     <item>
      <widget class="QLabel" name="origFpsLabel">
       <property name="text">
        <string>Original:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="origFps">
       <property name="text">
        <string>&lt;span style=&quot; font-weight:600;&quot;&gt;12 fps&lt;/span&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="newFpsLabel">
       <property name="text">
        <string>New:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="newFpsBox">
       <property name="suffix">
        <string> fps</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
       <property name="value">
        <number>12</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="speedTitle">
     <property name="text">
      <string>&lt;span style=&quot; text-decoration: underline;&quot;&gt;Speed&lt;/span&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="speedGroup">
     <item>
      <widget class="QLabel" name="origSpeedLabel">
       <property name="text">
        <string>Original:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="origSpeed">
       <property name="text">
        <string>&lt;span style=&quot; font-weight:600;&quot;&gt;1.00x&lt;/span&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="newSpeedLabel">
       <property name="text">
        <string>New:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="newSpeedBox">
       <property name="suffix">
        <string>x</string>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>90.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="warningLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:red;&quot;&gt;Warning: Retiming to a lower fps or higher speed may result in the loss of frames. This action cannot be undone.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RetimeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RetimeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
